src/lib/slurpParser.js

```diff
@@ -0,0 +1,30 @@
+import e from 'gEngine/engine'
+
+const MAX_WIDTH = 8
```

Comment:
Maybe rename this MAX_COLUMN, with a comment to say that this is a usability feature so that lots of SIPs in a SLURP wouldn't be frustratingly long?